Documentation/contributing/development/reviewers_committers/review_vendor.rst

```diff
@@ -0,0 +1,117 @@
+repositories:
+
+* `cilium/cilium <cilium_slash_cilium_reviews_>`_
+* `cilium/hubble <cilium_slash_hubble_reviews_>`_
+* `cilium/cilium-cli <cilium_slash_cilium_cli_reviews_>`_
+* `cilium/certgen <cilium_slash_certgen_reviews_>`_
+
+Open Pull Requests awaiting reviews from @cilium/vendor are listed by repository
+through the previously listed links.
+
+To join the team, you must be a Cilium Reviewer. see `Cilium's Contributor
+Ladder <ladder_>`_ for details on the requirements and the application process.
+
+.. _vendor_team: https://github.com/orgs/cilium/teams/vendor
+.. _go_dot_mod: https://github.com/cilium/cilium/blob/main/go.mod
+.. _go_dot_sum: https://github.com/cilium/cilium/blob/main/go.sum
+.. _vendor_slash: https://github.com/cilium/cilium/blob/main/vendor
+.. _cilium_slash_cilium_reviews: https://github.com/cilium/cilium/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+team-review-requested%3Acilium%2Fvendor
+.. _cilium_slash_hubble_reviews: https://github.com/cilium/hubble/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+team-review-requested%3Acilium%2Fvendor
+.. _cilium_slash_cilium_cli_reviews: https://github.com/cilium/cilium-cli/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+team-review-requested%3Acilium%2Fvendor
+.. _cilium_slash_certgen_reviews: https://github.com/cilium/certgen/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+team-review-requested%3Acilium%2Fvendor
+.. _ladder: https://github.com/cilium/community/blob/main/CONTRIBUTOR-LADDER.md
+
+Reviewing Pull Requests
+=======================
+
+This section describes some of the process and expectations for reviewing PRs
+on behalf of @cilium/vendor. Note that :ref:`the generic PR review
+process for Committers <review_process>` still applies, even though it is not
+specific to dependencies.
+
+Existing Dependencies
+---------------------
+
+Updates to existing dependencies most commonly occur through PRs opened by
+`Renovate <renovate_>`_, which is a 3rd party service used throughout the
+Cilium organization. Renovate continually checks repositories for out-of-date
+dependencies and opens new PRs to update any it finds.
+
+.. note::
+   Renovate is configured via a file named ``renovate.json5`` found within the
+   ``.github/`` folder of each repository it is enabled on. Cilium's Renovate
+   configuration is long and complex, having been developed to enable
+   dependency updates to Docker images and GitHub Actions. Only a few pieces
+   of the configuration are relevant to the @cilium/vendor team.
+
+.. _renovate: https://docs.renovatebot.com
+
+When reviewing PRs that update an existing dependency, members of the
+@cilium/vendor team are required to ensure that the update does not include
+any breaking changes or licensing issues. These checks are facilitated via
+GitHub Action CI workflows, which are triggered by commenting ``/test`` within
+a PR. See :ref:`CI  / GitHub Actions <ci_gha>` for more information on their
+use.
+
+New Dependencies
```

Comment:
I see where you're coming from. Would you like to bring this up in a Cilium community meeting? That would be a really good place for discussion.

Reply:
Sure, I can bring this up in a community meeting, unfortunately I won't be able to join the one on 23, but happy to discuss it on 30.

Reply:
In general I tend to advocate for creating some clear public discussion format (such as a PR or issue) where community members can asynchronously discover the topic and engage in discussion. This should be the primary forum, which can be enhanced with additional conversation in other forums such as Slack or synchronous meetings.

If @cilium/vendor wish to run a monthly meeting for coordinating things like this or other topics, you are welcome to do so. @xmulligan or I can help with providing recorded meetings and calendar entries for this. At the same time, I also want to highlight that synchronous meetings do not fit all contributors - for instance, contributors may be in a timezone that doesn't fit with other contributors to the group (classicly, you can pick 2/3 out of Americas / EMEA / Asia-Pacific for participation, and the time will be inconvenient for the third). Also some people struggle more with actively engaging in live conversation vs. taking time to process and form an opinion / response. From that perspective, if it helps to drive consensus then feel free to meet up. Please also ensure that the relevant conversation is replicated to a public GitHub issue/PR/discussion. For extra awareness you could share such links in the relevant Slack channels.

Reply:
Specifically for new dependencies, I think it's quite rare and I doubt that this would provide sufficient discussion to sustain a monthly meeting. But if it's just one topic in a range of topics then we could of course set something like that up.

Reply:
Another idea could be creating an issue template where people can ask to add new dependencies and @cilium/vendor could then review. You can see this example for requesting a new repo https://github.com/cilium/community/issues/new?template=new-repository.yaml. Not sure if this is too heavy weight though.

Reply:
I'm going to resolve this conversation for now so we can get this PR merged. Let's take the discussion to the community slack.